php/WP_CLI/Dispatcher/Subcommand.php

```diff
@@ -148,6 +148,10 @@ private function prompt_args( $args, $assoc_args ) {
 		foreach( $spec as $key => $spec_arg ) {
 
 			$current_prompt = ( $key + 1 ) . '/' . count( $spec ) . ' ';
+			$description_match = preg_match( '/' . preg_quote( $spec_arg['token'] ) . '\r?\n\:\s(.*)/', $this->longdesc, $matches );
```

Comment:
`$this->docparser->get_arg_desc( $spec_arg['name'] );` should return argument description, no?